common/src/main/java/tachyon/security/authorization/FileSystemAction.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security.authorization;
+
+/**
+ * POSIX style file system actions, e.g. read, write, etc.
+ */
+public enum FileSystemAction {
+  NONE("---"),
+  EXECUTE("--x"),
+  WRITE("-w-"),
+  WRITE_EXECUTE("-wx"),
+  READ("r--"),
+  READ_EXECUTE("r-x"),
+  READ_WRITE("rw-"),
+  ALL("rwx");
+
+  /** Symbolic representation */
+  private final String mSymbol;
+
+  /** Retain reference to value array. */
+  private static final FileSystemAction[] SVALS = values();
+
+  FileSystemAction(String s) {
+    mSymbol = s;
+  }
+
+  public String getSymbol() {
+    return mSymbol;
+  }
+
+  /**
+   * Check whether this action implies the passed-in action.
```

Comment:
s/Check/Checks 

Also, for consistency with the rest of our code base, add an empty line between the method description and the annotations:

```
Checks ...

@ param ...
@ return ...
```